autorest/azure/async_test.go

```diff
@@ -1106,6 +1106,6 @@ func makeLongRunningOperationErrorString(code string, message string) string {
 }
 
 func makeRequestErrorString(code, message string, status int) string {
-	return fmt.Sprintf("azure: Service returned an error. Code=%q Message=%q Status=%d",
+	return fmt.Sprintf("autorest/azure: Service returned an error. Code=%q Message=%q Details=<nil> Status=%d",
```

Comment:
It is used only once so I will remove this function and use deepEqual or string match instead.